include/mc_tasks/lipm_stabilizer/StabilizerTask.h

```diff
@@ -744,13 +744,18 @@ struct MC_TASKS_DLLAPI StabilizerTask : public MetaTask
     std::string surfaceName;
   };
 
-  /** @brief Compute the CoM offset and the sum wrench from the external wrenches.
+  /** @brief Compute the CoM offset (\alpha) and the ZMP coefficient (\kappa) and the sum wrench from the external
+   * wrenches. see Murooka et al. RAL 2021 eq (8)
    *
    *  @tparam TargetOrMeasured Change depending on the used wrenches
-   *  @param robot Robot used to transform surface wrenches (control robot or real robot)
+   *  @param robot [in] - Robot used to transform surface wrenches (control robot or real robot)
+   *  @param offset_alpha [out] - Com offset
+   *  @param coef_alpha [out] - ZmP coefficient
    */
   template<sva::ForceVecd ExternalWrench::*TargetOrMeasured>
-  Eigen::Vector3d computeCoMOffset(const mc_rbdyn::Robot & robot) const;
```

Comment:
@mmurooka, this function disappears and it is the only change of API this PR introduces, can you confirm that you don't use this function anywhere else?

Reply:
It is replaced with 
```  
template<sva::ForceVecd ExternalWrench::*TargetOrMeasured>
  void computeWrenchOffsetAndCoefficient(const mc_rbdyn::Robot & robot,
                                         Eigen::Vector3d & offset_alpha,
                                         double & coef_kappa) const;
```

Reply:
Oh true! Sorry...